### ISSUE TITLE
issues installing and running the cli

### ISSUE THREAD

**Issue by BigBlockStudios** [zurb/foundation-cli#151]
Completely frustrated after an entire day trying to get a couple foundation projects running ...  I can get the cli installed but foundation commands won't run, nor will it create a new project. Here is what I am doing:  (WSL Linux Ubuntu 18) 

```
sean@Panzer:/var/www/vhosts$ which foundation node gulp yarn bower ; foundation -v ; node -v ; npm -v ; foundation new --framework sites --template panini --directory foobar
/home/sean/.nvm/versions/node/v14.18.3/bin/foundation
/home/sean/.nvm/versions/node/v14.18.3/bin/node
/home/sean/.nvm/versions/node/v14.18.3/bin/gulp
/home/sean/.nvm/versions/node/v14.18.3/bin/yarn
/home/sean/.nvm/versions/node/v14.18.3/bin/bower
Foundation CLI version 2.3.2
v14.18.3
6.14.15


Downloading the project template...
Done downloading!

Installing dependencies...

undefined

There were some problems during the installation.

 ✓ New project folder created.
 ✗ Node modules not installed. Try running yarn install manually.

Once you've resolved the above issues, run foundation watch while inside the foobar folder.
```

no idea why it does not install the modules, but I can do it manually using either npm install or yarn install:

```
sean@Panzer:/var/www/vhosts/foobar$ yarn install
yarn install v1.22.17
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
Done in 16.14s.
sean@Panzer:/var/www/vhosts/foobar$ foundation watch
(node:28400) UnhandledPromiseRejectionWarning: TypeError: npm.commands.start.apply is not a function
    at /home/sean/.nvm/versions/node/v14.18.3/lib/node_modules/foundation-cli/lib/commands/watch.js:26:24
(Use node --trace-warnings ... to show where the warning was created)
(node:28400) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag --unhandled-rejections=strict (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:28400) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
sean@Panzer:/var/www/vhosts/foobar$ npm start

> foundation-zurb-template@1.0.0 start /var/www/vhosts/foobar
> gulp

[18:31:33] Requiring external module @babel/register
{
  html: [ 'dist/**/*.html' ],
  timeout: 1000,
  ignore: [ '.foundation-mq', /\.is-\w+/ ]
}
[18:31:35] Using gulpfile /var/www/vhosts/foobar/gulpfile.babel.js
[18:31:35] Starting 'default'...
[18:31:35] Starting 'build'...
[18:31:35] Starting 'clean'...
[18:31:35] Finished 'clean' after 2.19 ms
[18:31:35] Starting 'pages'...
[18:31:35] Starting 'javascript'...
[18:31:35] Starting 'images'...
[18:31:35] Starting 'copy'...
[18:31:35] Finished 'images' after 335 ms
[18:31:35] Finished 'pages' after 562 ms
[18:31:35] Finished 'copy' after 569 ms
[18:31:39] asset app.js 720 KiB [emitted] (name: app) 1 related asset

webpack 5.61.0 compiled successfully
[18:31:39] Finished 'javascript' after 4.61 s
[18:31:39] Starting 'sassBuild'...
[18:31:42] Finished 'sassBuild' after 2.17 s
[18:31:42] Starting 'styleGuide'...
[18:31:42] Finished 'styleGuide' after 28 ms
[18:31:42] Finished 'build' after 6.81 s
[18:31:42] Starting 'server'...
[Browsersync] Access URLs:
 -------------------------------------
       Local: http://localhost:8000
    External: http://172.19.59.58:8000
 -------------------------------------
          UI: http://localhost:3001
 UI External: http://localhost:3001
 -------------------------------------
[Browsersync] Serving files from: dist
[18:31:42] Finished 'server' after 129 ms
[18:31:42] Starting 'watch'...

```

First off changing the template from "zurb' to 'panini' and NOT documenting it was extremely annoying (and cost about an hour fiddling with git ~ foundation new --framework sites --template zurb --directory foobar ~ fails with a vague git not setup correct error)

Second, does the CLI work at all for executing the watch & build commands ?? 

I've tried this using nvm to install node 10,12,14 & 16 (14 was the only one I got this far with)  I've removed node, bower & gulp system packages that were installed using apt so everything should be using the nvm profiles/packages.. 

What is the actual problem here and how do I fix it??  Is there updated cli installatioin documentation that actually lists known working versions of node/bower/gulp etc?

**Comment by joeworkman** [comment]
The CLI was completely broken. I released v2.3 that at least fixed the new command. Node 14 is the only version of node that it works with at this time. 

In the future, I do not think this cli will be developed any further. I have started a new tool called Proton that is a replacement for Panini. It also has an init feature that will be able to start a new project from a template similar to this cli.
